cirq-core/cirq/contrib/quimb/mps_simulator.py

```diff
@@ -219,59 +220,67 @@ def _simulator_state(self):
 
 
 @value.value_equality
-class MPSState(ActOnArgs):
-    """A state of the MPS simulation."""
+class _MPSQuantumState:
```

Comment:
The added "Quantum" doesn't meaningfully distinguish this from `MPSState`, since both are "quantum" to some level of interpretation. Maybe "_MPSHandler", since this is the part that actually performs the MPS-level operations?